### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by mpaglia0** [eoyilmaz/displaycal-py3#317] [Q&A]
Hi!
I would like to contribute with translations but cannot understand how yaml files can be translated... I usually use .po

Thank you,
Maurizio

**Comment by eoyilmaz** [comment]
Very straightforward to be honest, just copy the `en.yaml` file and rename it to your language and then translate all teh text after the `|-` sign, the ones between quotes (") should stay as they are, they are not to be translated that is.

**Comment by mpaglia0** [comment]
Thanks
